### DISCUSSION TITLE
Unify the product typeface

### DISCUSSION THREAD

**Discussion by zezorzan** [outline/outline#1295] [General] [answered]
Hello!

I realized that the product uses the OS typeface 99% of the time. But in a few situations, **Lato** appears, in the headings of a document (not the title, though) and in the titles of the pages, for example.

Would it be possible to unify everything to the OS typeface? For me as a user, it seems a little inconsistent.

Thanks in advance!

**Comment by tommoor** [comment, marked as answer]
Hey @zezorzan – yep this sounds like something we should do. The typeface in the headings kind of snuck through from the marketing site which I'm in the final stages of completely disconnecting from the OSS project.

**Reply by zezorzan** [reply]
Cool, thank you so much!

**Reply by tommoor** [reply]
This was done in v0.44.0 btw

**Reply by zezorzan** [reply]
Thank you!
